packages/react-dev-utils/InterpolateHtmlPlugin.js

```diff
@@ -15,7 +15,6 @@
 // Learn more about creating plugins like this:
 // https://github.com/ampedandwired/html-webpack-plugin#events
 
-'use strict';
```

Comment:
I'm confused why it thinks it's a module. We're not using `import` syntax anywhere.